modules/ROOT/pages/functions/user_guide/templates-function-kn.adoc

```diff
@@ -1,12 +1,32 @@
 = Using {short-name} templates
 
 The {product-name} templates provided are designed so that developers can concentrate solely on the business logic of their applications.
-Tasks such as managing network connections are managed by the runtime framework of the template.
 
+The template provided for your created function will be based on which trigger type is specified when creating the function. For example, if you specify `trigger: http` when creating the function, the template provided will be tailored to support HTTP use cases. If you specify `trigger: events`, the template will be tailored to better suit https://cloudevents.io/[CloudEvent] use cases.
```

Comment:
```suggestion
The template provided for your created function will be based on which trigger type is specified when creating the function. For example, if you specify `--trigger http` when creating the function, the template provided will be tailored to support HTTP use cases. If you specify `--trigger events`, the template will be tailored to better suit https://cloudevents.io/[CloudEvent] use cases.
```

Reply:
@lance I wouldn't spend too much time reviewing these since it's super WIP at the minute :smile: but this is helpful, thank you